This is a workaround for Alpine Linux 3.20 (and older) where we don't
have the no-namespace version of docbook-xsl needed for building
PostgreSQL man pages, so we have to use prebuilt man pages distributed
in the tarball as it has been so far.

--- a/doc/src/sgml/Makefile
+++ b/doc/src/sgml/Makefile
@@ -26,7 +26,7 @@
 include $(top_builddir)/src/Makefile.global
 
 
-all: man
+all:
 
 distprep: distprep-man
 
@@ -236,7 +236,7 @@
 
 ## Install man
 
-install-man: man installdirs
+install-man: installdirs
 
 sqlmansect ?= 7
 sqlmansectnum = $(shell expr X'$(sqlmansect)' : X'\([0-9]\)')
